### ISSUE TITLE
logs/keyfreq_...txt

### ISSUE THREAD

**Issue by sshleifer** [karpathy/ulogme#14]
My key freq log file has timestamps, but each timestamp has a zero associated with it. When I load the webpage, there is no number next to total number of key strokes: The other log file looks reasonable. 

➜  ulogme git:(master) cat logs/keyfreq_1417262400.txt
1417318604        0
1417318613        0
1417318622        0
1417318631        0
1417318640        0
....

**Comment by 10amc-2** [comment]
I'm having the same issue, is this still unresolved?

**Comment by andymaheshw** [comment]
Yes, the trick is to start the script, go to accessibility, you will see iTerm, add it. Stop the script, and restart again.
